src/test/java/uk/gov/hmcts/probate/validator/IHTValidationRuleTest.java

```diff
@@ -73,16 +74,34 @@ public void testValidateWithSuccessWhenIhtIsNull() {
     }
 
     @Test
-    public void testValidateFailureWhenNetHigherThanGross() {
+    public void testValidateFailureWhenProbateNetHigherThanGross() {
         when(inheritanceTaxMock.getGrossValue()).thenReturn(LOWER_VALUE);
         when(inheritanceTaxMock.getNetValue()).thenReturn(HIGHER_VALUE);
-        when(businessValidationMessageService.generateError(BUSINESS_ERROR, IHT_NET_GREATER_THAN_GROSS))
+        when(businessValidationMessageService.generateError(BUSINESS_ERROR, IHT_PROBATE_NET_GREATER_THAN_GROSS))
                 .thenReturn(businessValidationError);
 
         List<FieldErrorResponse> validationError = underTest.validate(ccdDataMock);
 
         assertThat(validationError.isEmpty(), is(false));
-        verify(businessValidationMessageService, times(1)).generateError(BUSINESS_ERROR, IHT_NET_GREATER_THAN_GROSS);
+        verify(businessValidationMessageService, times(1))
+            .generateError(BUSINESS_ERROR, IHT_PROBATE_NET_GREATER_THAN_GROSS);
+        assertTrue(validationError.contains(businessValidationError));
+    }
+
+    @Test
+    public void testValidateFailureWhenIHTNetHigherThanGross() {
+        when(inheritanceTaxMock.getGrossValue()).thenReturn(LOWER_VALUE);
+        when(inheritanceTaxMock.getNetValue()).thenReturn(HIGHER_VALUE);
+        when(inheritanceTaxMock.getIhtEstateGrossValue()).thenReturn(LOWER_VALUE);
+        when(inheritanceTaxMock.getIhtEstateNetValue()).thenReturn(HIGHER_VALUE);
+        when(businessValidationMessageService.generateError(BUSINESS_ERROR, IHT_ESTATE_NET_GREATER_THAN_GROSS))
+            .thenReturn(businessValidationError);
+
+        List<FieldErrorResponse> validationError = underTest.validate(ccdDataMock);
+
+        assertThat(validationError.isEmpty(), is(false));
+        verify(businessValidationMessageService, times(1))
+            .generateError(BUSINESS_ERROR, IHT_ESTATE_NET_GREATER_THAN_GROSS);
         assertTrue(validationError.contains(businessValidationError));
     }
 }
```

Comment:
Worth adding a test if Gross and net values are equal for both probate and iht values